src/extension/serializer.ts

```diff
@@ -291,8 +292,10 @@ export class GrpcSerializer extends SerializerBase {
   protected ready: ReadyPromise
 
   static async initGrpc(): Promise<ParserServiceClient> {
+    const config = workspace.getConfiguration('runme.server')
```

Comment:
I'd suggest to move all the config parsing logic into a helper/utility that returns a validated set of values which can be used wherever this config's required

Reply:
Moved to a configuration utility plus zod schema validation